content/blog/mirai-2-5-0/index.Rmd

```diff
@@ -0,0 +1,227 @@
+
+![*Illustrative span structure shown in a Jaeger collector UI*](otel-screenshot.png)
+
+The hierarchical tracing structure allows you to track the flow from initiation through to final evaluation, enabling precise performance analysis and bottleneck identification.
+
+### Reproducible parallel RNG
+
+We're equally delighted to announce reproducible parallel random number generation. Developed in consultation with our tidymodels colleagues and core members of the mlr3 team, this is a great example of the R community pulling together to implement crucial infrastructure.
+
+Technically this feature has been around since the recent 2.4.1 release, but it solves a long-standing challenge in parallel computing in R. We hope that by announcing it here, it's put to good use for reproducible science.
+
+mirai has long used L'Ecuyer-CMRG streams (since prior to v1.0) for statistically-sound parallel RNG. Streams essentially cut into the RNG’s period (a very long sequence of pseudo-random numbers) at intervals that are far apart from each other that they do not in practice overlap. This ensures that statistical results obtained from parallel computations remain correct and valid. 
+
+Now, we offer two modes:
+
+**Default behaviour (`seed = NULL`)**: Creates independent streams for each daemon, matching the behaviour of base R's parallel package, ensuring statistical validity but not numerical reproducibility between runs.
+
+**Reproducible mode (`seed = integer`)**: Creates a stream for each `mirai()` call rather than each daemon, guaranteeing identical results across runs regardless of the number of daemons used.
+
+```{r}
+with(
+  daemons(3, seed = 1234L),
+  mirai_map(1:3, rnorm, .args = list(mean = 20, sd = 2))[]
+)
+
+# Run again - identical results!
+```
+
+### User interface improvements
+
+New scoped helper functions `with_daemons()` and `local_daemons()` make working with compute profiles much more convenient by allowing the temporary switching of contexts:
+
+```r
+# Work with specific compute profiles
+with_daemons("gpu", {
+  result <- mirai(gpu_intensive_task())
+})
+
+# Local version for use inside functions
+async_gpu_intensive_task <- function() {
+  local_daemons("gpu")
+  mirai(gpu_intensive_task())
+}
+```
+
+The `daemons()` function has been redesigned to be more ergonomic.
+Creating new daemons automatically resets existing ones (no more manual `daemons(0)` calls):
+
+```r
+# Old approach
+daemons(0)  # Had to reset first
+daemons(4)
+
+# New approach - automatic reset
+daemons(4)  # Just works, resets if needed
+```
+
+
+### Developer features
+
+A new function `info()` provides a more succinct alternative to `status()` for reporting key statistics. This is now optimized and available for programmatic use.
+
+```{r, eval = FALSE}
+info()
+#> connections  cumulative    awaiting   executing   completed 
+#>           4           4           8           4           2
+```
+
+## Powering the R ecosystem
```

Comment:
I think you could drop this section; it's not clear to me that it adds anything.